client/app/dbaas/logs/detail/options/logs-options.service.js

```diff
@@ -31,7 +31,7 @@ class LogsOptionsService {
      * @memberof LogsOptionsService
      */
     getOptions (serviceName) {
-        const transformOption = this.transformOption.bind(this);
+        const transformOption = this._transformOption.bind(this);
```

Comment:
I've made another comment about the bind.  You should be able to do the same thing here

Reply:
removed bind